docs/public/index.html

```diff
@@ -1,22 +0,0 @@
-<!--
-  Copyright (c) 2018-present, GM Cruise LLC
-
-  This source code is licensed under the Apache License, Version 2.0,
-  found in the LICENSE file in the root directory of this source tree.
-  You may not use this file except in compliance with the License.
--->
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
-    <meta name="theme-color" content="#000000" />
-    <title>webviz</title>
-    <script>
-      window.location = "worldview";
-    </script>
-  </head>
-  <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
-  </body>
-</html>
```

Comment:
Removed this file in favour of another symlink. We should get a nicer landing page here though.